include/cuco/detail/static_set/static_set.inl

```diff
@@ -337,6 +337,29 @@ void static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>
   impl_->find_async(first, last, output_begin, ref(op::find), stream);
 }
 
+template <class Key,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+template <typename InputIt, typename ProbeEqual, typename ProbeHash, typename OutputIt>
+void static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::find_async(
+  InputIt first,
+  InputIt last,
+  ProbeEqual const& probe_equal,
+  ProbeHash const& probe_hash,
+  OutputIt output_begin,
+  cuda::stream_ref stream) const
```

Comment:
```suggestion
void static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::find_async(
  InputIt first,
  InputIt last,
  OutputIt output_begin,
  cuda::stream_ref stream,
  ProbeEqual const& probe_equal,
  ProbeHash const& probe_hash) const
```

Would this reordering make sense? We could default `probe_equal` and `probe_hash` to the ones the container currently uses. This would save us one more overload.

Reply:
We arrange arguments in the order from input to output, with the stream parameter always placed last and given a default value.